bw2io/extractors/ecospold2.py

```diff
@@ -233,6 +336,18 @@ def extract_activity(cls, dirpath, filename, db_name):
 
     @classmethod
     def abort_exchange(cls, exc, comment=None):
+        """
+        Set the uncertainty type of the input exchange to UndefinedUncertainty.id. Remove the keys "scale", "shape", "minimum", and "maximum" from the dictionary. 
+        Update the "loc" key to "amount". Append "comment" to "exc['comment']" if "comment" is not None, 
+        otherwise append "Invalid parameters - set to undefined uncertainty." to "exc['comment']".
+
+        Args:
+            exc (dict): The input exchange.
+            comment (str, optional): A string to append to "exc['comment']". Defaults to None.
```

Comment:
...same here: https://github.com/brightway-lca/brightway2-io/pull/176/files#r1157006084

Reply:
Edited to conform to standards.